src/app/langages/sql.ts

```diff
@@ -1,8 +1,9 @@
 import {EditorView} from "@codemirror/basic-setup";
 import {HighlightStyle, tags} from "@codemirror/highlight";
 import {CodeMirrorTheme} from "../types";
-import {sql} from "@codemirror/lang-sql";
+import {StandardSQL} from "@codemirror/lang-sql";
 import {type Sqlite3Static, type DatabaseApi}  from "@sqlite.org/sqlite-wasm";
+import {LanguageSupport} from "@codemirror/language";
```

Comment:
Could we try and keep these imports together? Does Optimise Imports do this for you?